packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5706,13 +5730,17 @@ export default class Meeting extends StatelessWebexPlugin {
               remoteQualityLevel: this.mediaProperties.remoteQualityLevel,
             },
           })
-          .then(() => {
-            if (startShare) {
-              return this.requestScreenShareFloor();
-            }
+          .then(() =>
+            this.enqueueMediaUpdate(MEDIA_UPDATE_TYPE.LAMBDA, {
+              lambda: async () => {
```

Comment:
what is the reason for the name lambda here?

Reply:
other media update types are for specific predefined operations like updateAudio, updateVideo, updateMedia, this one is a generic one that just takes a function that can run anything, usually it's an anonymous function, so a "lambda function"